apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -4259,13 +4333,25 @@ sc_ws_oracle_contract_(Owner, GetVolley, ConnPid1, ConnPid2,
     % owner posts the correct
     CallContract(Owner, <<"place_bet">>, <<"\"", Answer/binary, "\"">>,
                   <<"string">>, <<"ok">>),
+    #{ active := true
+     , ct_version := 1
+     , deposit := 10
+     , id := ContractId
+     , owner_id := OwnerId
+     , referrer_ids := []
+     , state := State4
+    } = sc_ws_get_contract_assert_equal(ConnPid1, ConnPid2, ContractPubKey, Config),
+    <<0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4>> = maps:get(<<2>>, State4),
```

Comment:
A comment on these `<<2>>` and related values ease readability of the test. Though debatable so ok.